pkg/operator/starter.go

```diff
@@ -62,7 +62,9 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		kubeClient,
 		kubeInformersForTargetNamespace,
 		kubeInformersForTargetNamespace.InformersFor(operatorclient.TargetNamespace),
-		operatorClient, resourceapply.NewKubeClientHolder(kubeClient).WithAPIExtensionsClient(apiExtensionsClient),
+		operatorClient,
+		certManagerInformers,
```

Comment:
nit : It is a good practice to add new vars at the last of the list to avoid any conflicts arising when there are multiple contributions on the same. for now, the suggestion can be ignored.